src/vehicle_func.h

```diff
@@ -115,59 +115,55 @@ const struct Livery *GetEngineLivery(EngineID engine_type, CompanyID company, En
 SpriteID GetEnginePalette(EngineID engine_type, CompanyID company);
 SpriteID GetVehiclePalette(const Vehicle *v);
 
-extern const uint32 _veh_build_proc_table[];
-extern const uint32 _veh_sell_proc_table[];
-extern const uint32 _veh_refit_proc_table[];
-extern const uint32 _send_to_depot_proc_table[];
+extern const StringID _veh_build_msg_table[];
+extern const StringID _veh_sell_msg_table[];
+extern const StringID _veh_refit_msg_table[];
+extern const StringID _send_to_depot_msg_table[];
```

Comment:
Maybe not for this PR, though... these tables and all the related functions in this file are only used once or not at all. Maybe move the table to the respective users? Or even use `STR_ERROR_CAN_T_BUY_TRAIN + v->type` instead at those location?

Reply:
Ignoring for this one (also the other GetCmd... things).